src/librustc_metadata/schema.rs

```diff
@@ -375,14 +373,37 @@ impl<'a, 'gcx> HashStable<StableHashingContext<'a>> for EntryKind<'gcx> {
             EntryKind::AssociatedType(associated_container) => {
                 associated_container.hash_stable(hcx, hasher);
             }
-            EntryKind::AssociatedConst(associated_container, qualif) => {
+            EntryKind::AssociatedConst(associated_container, qualif, _) => {
```

Comment:
This ignores the last field, but `EntryKind::Const` above doesn't. cc @michaelwoerister